modin/pandas/base.py

```diff
@@ -1905,7 +1905,6 @@ def mask(
                 level=level,
                 errors=errors,
                 try_cast=try_cast,
-                squeeze_self=not self._is_dataframe,
```

Comment:
Why was this argument removed?

Reply:
I'm assuming this is because `mask` and `pct_change` are equivalent in pandas whether applied to a Series or a 1 col DataFrame?

Reply:
yeah, they seem to behave identically in pandas